app/styles/modules/_home.scss

```diff
@@ -0,0 +1,202 @@
+#landing-page {
+  opacity: 0;
+}
+
+/* The following is a lightly modified version of the styles
+*  used by the fullpage plugin.
+*/
+
+#superContainer {
+  height: 100%;
+  position: relative;
+
+/* Touch detection for Windows 8 */
+-ms-touch-action: none;
```

Comment:
Also, indenting is a bit messed up in some of these styles.
> [W] Indentation: Line should be indented 2 spaces, but was indented 0 spaces

Actually, I can do a separate PR to test and remove vendor prefixes, sort, and tweak spacing to appease the scss-lint gods.